### ISSUE TITLE
Re-index after update to ArcLight 1.1 is completed

### ISSUE THREAD

**Issue by corylown** [sul-dlss/vt-arclight#627]
Blocked by #608 

This can be done at any time after #608 is completed. The changes in #607 are compatible with the current index produced from ArcLight 1.0 or an updated index created from ArcLight 1.1.

Tasks:

- [ ] Test re-indexing on vt-arclight stage.
- [ ] Sign off from PO (Lauren Sorensen) that the app looks fine with the update index on stage.
- [ ] Run re-indexing on production.
- [ ] Remove code that is no longer needed to support ArcLight 1.0 index
  - [ ] Remove override of `Arclight::Parent` to use the default behavior from ArcLight 1.1
  - [ ] Remove `attribute :parent_ids, Blacklight::Types::Array, 'parent_ssim'` from SolrDocument to use the default configuration from ArcLight 1.1
  - [ ] Change any calls to `SolrDocument.global_id` to `SolrDocument.id` (all currently in component templates): https://github.com/search?q=repo%3Asul-dlss%2Fvt-arclight%20global_id&type=code

**Comment by corylown** [comment]
@dnoneill I don't plan on doing any virtual tribunals work during the upcoming arclight work cycle. This could happen during this maintenance week. It's also more of a housekeeping task and not urgent, although it will be a blocker to upgrading to Arclight 2.0 whenever that is released. The re-indexing on production may require some downtime and coordination with Lauren to find a good time to do that. I can't remember how long the re-indexing takes.

**Comment by marlo-longley** [comment]
@dnoneill VT is special because it's all one big collection file.
See this PR:
https://github.com/sul-dlss/vt-arclight/pull/460/files

I think the `vt:index` task can be run after making sure we have the latest version of the collection file on the server.
https://github.com/sul-dlss/vt-arclight/blob/d8f5cf9a4a34f62ac02d0bab1ba09e8bbbd4a9cc/lib/tasks/vt.rake#L30
